packages/node_modules/samples/browser-meetings/index.html

```diff
@@ -144,88 +180,98 @@ <h2>Meeting Controls</h2>
+            <label id="sd-video-input-device-status"></label>        
           </div>
         </fieldset>
       </form>
 
       <br />
 
-      <form id="sending-sources-controls">
+      <form id="toggling-sources-controls">
         <fieldset>
-          <legend>Sending Sources</legend>
-          <div>
-            <button id="source-controls-send-audio" type="button">Toggle Sending Audio</button>
-            <label id="source-controls-sending-audio">[[SENDING AUDIO STATE]]</label>
-          </div>
-          <div>
-            <button id="source-controls-send-video" type="button">Toggle Sending Video</button>
-            <label id="source-controls-sending-video">[[SENDING VIDEO STATE]]</label>
-          </div>
-          <div>
-            <button id="source-controls-send-screenshare" type="button">Toggle Sending Screenshare</button>
-            <label id="source-controls-sending-screenshare">[[SENDING SCREENSHARE STATE]]</label>
-          </div>
+          <legend>Toggling Sources</legend>
           <div>
-            <select id="source-controls-sending-qualities">
-              <option value="LOW">Low</option>
-              <option value="MEDIUM">Medium</option>
-              <option value="HIGH">High</option>
-            </select>
-            <button id="source-controls-set-sending-quality" type="button">Set Quality</button>
-            <label id="source-controls-sending-quality">[[SENDING QUALITY STATE]]</label>
+            <label class="context-info">Toggle Sending Audio</label>
+            <button id="ts-toggle-audio" type="button" onclick="toggleSendAudio()">meeting.unmuteAudio() / meeting.muteAudio()</button>
+            <label id="ts-toggle-audio-status"></label>
           </div>
-        </fieldset>
-      </form>
-
-      <br />
-
-      <form id="receiving-sources-controls">
-        <fieldset>
-          <legend>Receiving Sources</legend>
           <div>
-            <button id="source-controls-receive-audio" type="button">Toggle Receiving Audio</button>
-            <label id="source-controls-receiving-audio">[[RECEIVING AUDIO STATE]]</label>
+            <label class="context-info">Toggle Sending Video</label>
+            <button id="ts-toggle-video" type="button" onclick="toggleSendVideo()">meeting.unmuteVideo() / meeting.muteVideo()</button>
+            <label id="ts-toggle-video-status"></label>
           </div>
           <div>
-            <button id="source-controls-receive-video" type="button">Toggle Receiving Video</button>
-            <label id="source-controls-receiving-video">[[RECEIVING VIDEO STATE]]</label>
+            <label class="context-info">Sending Screenshare</label>
+            <button id="ts-start-screenshare" type="button" onclick="startScreenShare()">meeting.updateShare()</button>
+            <button id="ts-stop-screenshare" type="button" onclick="stopScreenShare()">meeting.stopShare()</button>
```

Comment:
For other reviewers, note that start/stop screenshare methods in the SDK samples have a pre-existing bug that makes the screen share fail to start/stop. This can be ignored for the scope of the associated task on this change request.